src/server/mod.rs

```diff
@@ -1,11 +1,8 @@
 //! The ipc-agent json rpc node.
 
-pub mod config;
 pub mod jsonrpc;
-mod request;
-mod response;
+pub mod request;
+pub mod response;
 
 const DEFAULT_JSON_RPC_SERVER_VERSION: &str = "2.0";
-pub const DEFAULT_JSON_RPC_SERVER_ENDPOINT: &str = "json_rpc";
-pub const DEFAULT_JSON_RPC_SERVER_PROTOCOL: &str = "http";
-pub const DEFAULT_JSON_RPC_SERVER_ADDR: &str = "127.0.0.1:3030";
```

Comment:
Where these variables moved somewhere?

Reply:
They are no longer needed, I removed them.

Reply:
I think these should be moved to the relevant module in the config crate or removed altogether. The latter option actually might make more sense because (1) we might want to force the user to be explicit about the server configuration and (2) we might not want to launch the server at all in some circumstances.

Reply:
@hmoniz For `DEFAULT_JSON_RPC_SERVER_VERSION` it might be better to hard code as a const. This is actually a strict requirement from the spec. Maybe I call it `JSON_RPC_VERSION`, make it sound less like config. `DEFAULT_JSON_RPC_SERVER_ENDPOINT` I can make it into a config parameter.

Reply:
BTW, maybe we can just hard code the `json_rpc` endpoint?

Reply:
Just different host and port? They call share the same `json_rpc` path right? Like `localhost:3030/json_rpc` and `localhost:3031/json_rpc`, but the `/json_rpc` part remains the same.

Reply:
Oh, agree. The `json_rpc` path can be hardcoded. Sorry for the misunderstanding.

Reply:
nice nice! I have updated accordingly already, resolving this.